examples/Kindle2Folder.rb

```diff
@@ -0,0 +1,29 @@
+require 'kindle_highlights'
```

Comment:
This is a great example! Thanks for adding it. Really nice to have a folder like this.

I'd like to follow the snake case convention for file names, so `Kindle2Folder.rb` becomes `kindle_to_folder.rb`.